File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/NoiTru_NhapVien.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NoiTru_NhapVien } from './NoiTru_NhapVien.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NoiTru_NhapVienService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'NhapVien_Id', 'TiepNhan_Id', 'NgayNhapVien', 'ThoiGianNhapVien', 'NoiNhapVien_Id', 'BacSiChiDinh_Id', 'LyDoNhapVien_Id', 'KhoaDieuTri_Id', 'BenhAn_Id', 'ChanDoan', 'ICD_ChanDoan', 'TrangThai', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'KhamBenh_Id', 'CapCuu', 'LoaiBenhAn_Id'];

    List: NoiTru_NhapVien[] | undefined;
    ListFilter: NoiTru_NhapVien[] | undefined;
    FormData!: NoiTru_NhapVien;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NoiTru_NhapVien";
    }
}

File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/NoiTru_NhapVienService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class NoiTru_NhapVienService : BaseService<NoiTru_NhapVien, INoiTru_NhapVienRepository>
    , INoiTru_NhapVienService
    {
    private readonly INoiTru_NhapVienRepository _NoiTru_NhapVienRepository;
    public NoiTru_NhapVienService(INoiTru_NhapVienRepository NoiTru_NhapVienRepository) : base(NoiTru_NhapVienRepository)
    {
    _NoiTru_NhapVienRepository = NoiTru_NhapVienRepository;
    }
    }
    }

